src/test/java/liquibase/ext/spanner/AbstractMockServerTest.java

```diff
@@ -258,23 +250,6 @@ private static void registerDefaultResults() {
                         0,
                         DatabaseMetaData.columnNoNulls)))));
 
-    // Register results for Liquibase locking.
```

Comment:
This is the one that caused the problem. This would be called when the first mock server test was executed, but if that was after the emulator tests had already been executed, it would mean that Liquibase had already been loaded and initialized. Registering a custom generator after Liquibase has been initialized is not possible.

Reply:
Yes -- that makes sense, and also why it was inconsistent. We could have separate tags and separate build targets in gradle so that emulator is run separately from unittest?

We'll go with this for now -- as long as it also works locally consistently, too.